frontend/src/pages/instance/Settings/HighAvailability.vue

```diff
@@ -66,8 +70,13 @@ export default {
         }
     },
     computed: {
+        ...mapState('account', ['team', 'features']),
         isHA () {
             return this.instance?.ha?.replicas !== undefined
+        },
+        haFeatureAvailable () {
+            const flag = this.team.type.properties.features?.ha
+            return flag === undefined || flag
```

Comment:
I believe this could be `flag ?? true`, but it's perhaps easier to read as it's written right now